client/app/components/service-status/service-status-action.controller.js

```diff
@@ -0,0 +1,66 @@
+import {
+  DEFAULT_TARGET,
+  RENEW_URL,
+} from './service-status-action.constant';
+
+export default class {
+  constructor(constants, OvhApiMe) {
+    'ngInject';
+
+    this.constants = constants;
+    this.OvhApiMe = OvhApiMe;
+  }
+
+  $onInit() {
+    if (
+      !angular.isObject(this.serviceInfos)
+      || !_.isString(this.serviceName)
+    ) {
+      throw new Error('serviceExpirationDate: Missing parameter(s)');
+    }
+
+    return this.getOrderUrl()
+      .finally(() => {
+        this.loading = false;
```

Comment:
`this.loading` is never `true`

Reply:
It depends if the cache is invalidated.

ref: https://github.com/ovh-ux/ovh-api-services/blob/master/src/me/me.v6.service.js#L14

Reply:
There was no `this.loading = true` before @antleblanc  :)

Reply:
There is: https://github.com/ovh-ux/ovh-manager-web/pull/897/files#diff-99c12ff95b212c49e70559194ed55bb9R21